### PR TITLE
Add support of block type

### PR DESCRIPTION
**Pull request by shoukoo**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -82,6 +73,26 @@ func main() {
...
 }
 
+// readConfig to read config and generate tasks
+func readConfig(config string) ([]*walker.Task, error) {
+	b, err := ioutil.ReadFile(config)
+	if err != nil {
+		return nil, fmt.Errorf("Can't find config file %v", err)
+	}
+
+	if len(files) == 0 {
```
Comment (alexlance) [inline main.go:83]: Maybe the "len(files)" validation shouldn't be in this function?

File: tf.yaml [hunk 2]
```diff
@@ -3,5 +3,6 @@ aws_instance:
     - Name
     - terraform
 
-  volume_tags:
+  lifecycle:
+    - create_before_destroy
```
Comment (alexlance) [inline tf.yaml:7]: Cool, can we add some more stuff to this? Like the wait_for_user_data thing and maybe the public_ip_address = false thing?

Comment (shoukoo) [reply tf.yaml:7]: This check can't verify values atm, it will take a long time to implement this feature.

Comment (alexlance) [reply tf.yaml:7]: Or maybe we can just add that extra stuff to our configuration file. Sorry I forgot which repo I was in :)

**Review by alexlance — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (shoukoo) [reply main.go:83]: We still need the condition there for me to test that part of code but I forgot to pass file variable as a function parameter.

**Pull request merged**